### PR TITLE
add libs and refactor

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Review by thomas-senechal — APPROVED** [review]
looks good to me

**Review by pivilartisant — APPROVED** [review]
lgtm !

**Review by peterjah — APPROVED** [review]

**Comment by Thykof** [comment]
I will publish it on npm package under an expe tag to no risk to break the whole lib.

File: cypress/component/integration/Clipboard.cy.tsx [hunk 1]
```diff
@@ -65,35 +65,6 @@ describe('Component | Integration | Clipboard', function () {
           'formatted content',
         );
       });
-
-      it('should fire the copy clipboard function', () => {
```
Comment (Thykof) [inline cypress/component/integration/Clipboard.cy.tsx:69]: revert commit that remove the clipboard dep and add it to optimize dep of vite config

File: .github/workflows/npm-publish-dev.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Npm nightly publish
 
 on:
   push:
-    branches: [main]
+    branches: [main, 428-include-a-lib-to-connect-wallets]
```
Comment (Thykof) [inline .github/workflows/npm-publish-dev.yml:5]: revert this before merge

File: scripts/publish-dev.sh [hunk 3]
```diff
@@ -3,14 +3,16 @@ set -e
 
 NPM_PACKAGE=$(cat package.json | jq -r '.name')
 
-npm version --preid dev --no-git-tag-version --no-commit-hooks prepatch
+TAG=next # change it back to dev when we are ready to publish to dev
```
Comment (Thykof) [inline scripts/publish-dev.sh:6]: change it to dev before merge

**Pull request merged**